benchmarking/deepcell-e2e/README.md

```diff
@@ -13,5 +13,5 @@ See also: [sample data README](https://github.com/dchaley/deepcell-imaging/tree/
 # Method
 
 - Setup a GCP Vertex AI managed notebook instance, connect to the instance using the link `OPEN JUPYTERLAB`
-- Run `pip install deepcell` in the VertexAI *terminal* prior to running the test notebook
+- Run `pip install deepcell --user` in the VertexAI *terminal* prior to running the test notebook
```

Comment:
Quick thing– this is incorrect actually, the requirements.txt is what needs to be installed. It'll all be in the notebook